test/IntegrationTests/StrongNamedTestsIntegrations.json

```diff
@@ -46,6 +46,28 @@
           "type": "OpenTelemetry.AutoInstrumentation.Instrumentations.Validations.MissingInstrumentationType",
           "action": "CallTargetModification"
         }
+      },
+      {
+        "caller": {},
+        "target": {
```

Comment:
Could you please extend
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/b0d4457610231e5e3a8af32f8856a9d0b0fbdf8d/tools/IntegrationsJsonGenerator/Program.cs#L68-L69
to programmatically add this instrumentation?
It will be removed if you execute the tool.

Reply:
[73f67ae](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1520/commits/73f67ae9744b3a2d54a39630c809a122f7d513e2)

Reply:
For now hard coding into the tool the integration tests that require mock instrumentations. Perhaps revisiting a more robust way of doing this.